### PR TITLE
RAC-249: Add Components dynamically in intro

### PR DESCRIPTION
**Pull request by ValentinMumble**
**Description (for Contributor and Core Developer)**

RAC-249: Add Components dynamically in intro

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: akeneo-design-system/src/Introduction.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        style={{
+          backgroundImage: `url("./all-visual-tsx-visual-tests-renders-components-${componentName.toLowerCase()}-standard-correctly-1-snap.png")`,
+        }}
+      ></div>
+      <span>
+        <strong>{componentName}</strong>
+        {Components[componentName].__docgenInfo.description}
+      </span>
+    </a>
+  ))}
```
Comment (tseho) [inline akeneo-design-system/src/Introduction.stories.mdx:149]: This is really good ... but what about the icons ?
We should have, ideally, all the icons in the same story, like if it was only one component with multiple styles.
Should this component, for storybook purposes, be in that list ?
Should we leave it like this for now and find an exception for it later ?

File: akeneo-design-system/src/all.visual.tsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ describe('Visual tests', () => {
       const root = await page.$('#root');
       if (null === root) return;
 
+      await new Promise(resolve => setTimeout(resolve, 500));
```
Comment (tseho) [inline akeneo-design-system/src/all.visual.tsx:34]: no better way to wait of full render ? :/

**Review by tseho — APPROVED** [review]

**Review by StevenVAIDIE — APPROVED** [review]
Nice job

File: akeneo-design-system/src/all.visual.tsx (hunk 2 above)
Comment (ValentinMumble) [reply akeneo-design-system/src/all.visual.tsx:34]: This is hacky but we didn't any other way 😞

File: akeneo-design-system/src/Introduction.stories.mdx (hunk 1 above)
Comment (ValentinMumble) [reply akeneo-design-system/src/Introduction.stories.mdx:149]: Icons will be in a dedicated Story yes and we can use the same workflow to display them dynamically, but the Introduction story only displays components in `src/components`

Comment (tseho) [reply akeneo-design-system/src/Introduction.stories.mdx:149]: perfect

**Pull request merged**